python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -527,27 +604,73 @@ def doDiffIm(processed_repo, sciencevisit, ccdnum, templatevisit, diffim_repo):
         config.doDecorrelation = True
 
     TODO: use coadds as templates by default, not another visit (DM-11422).
+    Some of the comments in this function are placeholders for DM-11422 work.
```

Comment:
This line should probably not be in the documentation, since it's a statement about the source code rather than the behavior.